README.md

```diff
@@ -81,6 +81,22 @@ Entries that are nested within another Entry. Only Categories or Sub-Categories
 }
 ```
 
+### Deprecated node mapping
+When breaking changes such as deletion/collapsing of IDs or moving to a different parent occur, the [`deprecated-node-mapping.json`](https://github.com/bugcrowd/vrt/deprecated-node-mapping.json) will serve as a reference to find the latest mapped ids so that deprecated nodes are not lost.
+
+#### Example
+_2 nodes being collapsed into 1_
+```json
+{
```

Comment:
In Contributing we only have examples with formats so far. I think that's fine, unless we want to add concrete examples for both.